api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java

```diff
@@ -58,22 +59,31 @@ public static RuntimeException convertGoogleIOException(IOException e) {
 
   public static RuntimeException convertFirecloudException(ApiException e) {
     log.log(e.getCode() >= 500 ? Level.SEVERE : Level.INFO, "Exception calling FireCloud", e);
+    if (e.getCause() instanceof SocketTimeoutException) {
```

Comment:
See the `is...Exception` pattern in this class. Since you do this twice, it would be nice to extract that to a similar boolean method like `isGoogleServiceUnavailableException`, etc.